tox.ini

```diff
@@ -25,3 +25,15 @@ commands = black -l 80 -t py36 --check \
            images \
            build.py
 deps = black
+
+[testenv:yamllint]
+description = check yaml with yamllint
+basepython = {[default]basepython}
+# yamllint doesn't handle helm templates well, in particular {{- bits, so we
+# exclude them in a previous part of the run in .github/workflows/lint.yaml by
+# switching to /tmp, where we copied and updated with sed, here it looks more
+# like we are actually running in the base dir, rather than displaying things
+# like /tmp/paws/values.yaml will show ./paws/values.yaml in output.
```

Comment:
This will not work locally if you try to run the tests (for what I can see, as the copy happens on github actions only), can we make sure the tests can be run locally too?

Reply:
I was thinking about this while creating it. As we can see I set it aside. My decision making was in that we don't have, to my knowledge, any instructions for how to run validation testing locally in the PAWS documentation. Though I agree it would be better if it was intuitive on how one might do that. I'll see if I can learn enough about tox to see if a tox only option is within normalish tox operation.

Reply:
Alrighty, I think this will run locally. Opened https://phabricator.wikimedia.org/T311549 to track adding a note to the README about running tox for local linting